src/app/opt/patch.rs

```diff
@@ -22,11 +14,10 @@ pub enum Patch {
 	},
 	Replace {
 		path: String,
-		value: Value,
+		value: String,
```

Comment:
Are you sure about this? Have you tried it?

Reply:
This is for type safety, the string will get auto converted into a surrealql value